ledger/src/blockstore_processor.rs

```diff
@@ -415,19 +393,19 @@ pub fn process_entries_for_tests(
 // Note: If randomize is true this will shuffle entries' transactions in-place.
 fn process_entries_with_callback(
     bank: &Arc<Bank>,
-    entries: &mut [EntryType],
-    randomize: bool,
+    entries: &[EntryType],
+    _randomize: bool,
     entry_callback: Option<&ProcessCallback>,
     transaction_status_sender: Option<&TransactionStatusSender>,
     replay_vote_sender: Option<&ReplayVoteSender>,
     transaction_cost_metrics_sender: Option<&TransactionCostMetricsSender>,
     timings: &mut ExecuteTimings,
     cost_capacity_meter: Arc<RwLock<BlockCostCapacityMeter>>,
+    replayer_handle: &ReplayerHandle,
 ) -> Result<()> {
-    // accumulator for entries that can be processed in parallel
-    let mut batches = vec![];
+    let mut executable_txs = vec![];
     let mut tick_hashes = vec![];
-    let mut rng = thread_rng();
+    // let mut rng = thread_rng();
 
     for entry in entries {
         match entry {
```

Comment:
need to review this more but i might be able to let it rip through ticks and register them all at the end. assume 64 ticks in slot, txs spaced evenly across ticks and 3k txs/s, 3000/64=max 47 txs

Reply:
err nvmd that doesn't make sense